test/Interop/Cxx/class/type-classification-non-trivial-silgen.swift

```diff
@@ -0,0 +1,25 @@
+// RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-sil %s | %FileCheck %s
+
+import TypeClassification
+
+// Make sure that "StructWithDestructor" is marked as non-trivial by checking for a
+// "destroy_addr".
+// CHECK-LABEL: @$s4main24testStructWithDestructoryyF
+// CHECK: [[AS:%.*]] = alloc_stack $StructWithDestructor
+// CHECK: destroy_addr [[AS]]
+// CHECK-LABEL: end sil function '$s4main24testStructWithDestructoryyF'
+public func testStructWithDestructor() {
+  let d = StructWithDestructor()
+}
+
+// Make sure that "HasMemberWithDestructor" is marked as non-trivial by checking
```

Comment:
`HasMemberWithDestructor` => `StructWithSubobjectDestructor`?

Reply:
@zoecarver ^^^

Reply:
Oops forgot about this. #33150.